src/Module/Api/Mastodon/Apps.php

```diff
@@ -46,8 +46,8 @@ public static function post(array $parameters = [])
 			DI::mstdnError()->RecordNotFound();
 		}
 
-		$client_id     = base64_encode(openssl_random_pseudo_bytes(32));
-		$client_secret = bin2hex(random_bytes(32));
+		$client_id     = bin2hex(openssl_random_pseudo_bytes(32));
+		$client_secret = bin2hex(openssl_random_pseudo_bytes(32));
```

Comment:
What was the rationale behind using the extension-dependent `openssl_random_pseudo_bytes()` rather than the PHP native `random_bytes()`?

Reply:
Using the native function would decrease our reliance on an extension. About the entropy, I trust @paragonie-scott and he's been advocating for `random_bytes()` including through a backward compatible version for PHP < 7.